Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1

```diff
@@ -93,7 +93,11 @@ Try{
     
     Write-Verbose "Executing SQLPackage.exe"  -Verbose
     
+    $ErrorActionPreference = 'Continue'
+
     Invoke-Expression -Command $SqlPackageCommand
+
+    $ErrorActionPreference = 'Stop'
```

Comment:
Seems fine for now. But we should follow-up with PowerShell team to understand why Error preference Stop is truncating the messages. Please let Pavan know about this.